src/state/ducks/student/selector.ts

```diff
@@ -0,0 +1,75 @@
+import { createSelector } from "reselect";
+import { IApplicationState } from "..";
+import { get, set } from "lodash";
+import { gradesPresidance } from "../../../helpers/data";
+
+const getList = (state: IApplicationState) => state.student.list;
+
+export const getStudentSummary = createSelector(getList, (list) => {
+  let Summary = {
```

Comment:
variable name should be small and as you are not changing summary reference it should be const rather than let.